start-docker-compose.sh

```diff
@@ -9,18 +11,19 @@ Help()
 {
    echo "Starts the ZAC Docker Compose environment using the 1Password CLI tools to retrieve secrets."
    echo
-   echo "Syntax: start-docker-compose.sh [-d|z|h]"
+   echo "Syntax: start-docker-compose.sh [-d|z|l|h]"
    echo "options:"
    echo "-d     Delete local Docker volume data before starting Docker Compose."
-   echo "-z     Also start the ZAC Docker container as part of the Docker Compose environment."
+   echo "-z     Also start last-known-good ZAC Docker container as part of the Docker Compose environment."
+   echo "-l     Start local ZAC image as part of Docker Compose environment"
```

Comment:
-z already does this if you use it correctly:  

```
./gradlew buildDockerImage && ZAC_DOCKER_IMAGE=ghcr.io/infonl/zaakafhandelcomponent:dev ./start-docker-compose.sh -z
```

Maybe : 'Build and start local ZAC Docker image..'?

Reply:
True, but the "correct" part is now coded in the script, so it should save a minute or two on every start.

Used your proposal for the help.